recipes/win_inet_pton/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{%set name = "win_inet_pton" %}
+{%set version = "1.0.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "a45bc3073469afb5afd0e395aaabd836040a20ba1f2a6dd1196fa2b210ba386e" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [not win]
+  number: {{ build_num }}
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - win_inet_pton
+
+about:
+  home: https://github.com/hickeroar/win_inet_pton
+  license: PUBLIC-DOMAIN
+  license_family: PUBLIC-DOMAIN
```

Comment:
Feel free to just drop `license_family`. Seems to cause more issues trying to fix this than just leaving it out.